docs/how-to/postgres-long-term-memory.mdx

```diff
@@ -0,0 +1,730 @@
+# Using PostgreSQL for Long-Term Memory
+
+CrewAI supports both SQLite and PostgreSQL as database backends for the long-term memory storage system. This guide explains how to configure and use PostgreSQL for storing agent memories, which is especially useful for production environments and when working with Amazon RDS or other managed database services.
+
+## Important Configuration Notes
+
+**Setting `memory=True` alone on a Crew will create SQLite-based memory storage by default.** To use PostgreSQL, you must explicitly configure it using one of the methods below. Simply having PostgreSQL environment variables set is not sufficient - you need to explicitly create and pass a PostgreSQL `LongTermMemory` instance to your crew.
+
+**For automatic memory saving to work, you need both `long_term_memory` AND `entity_memory` configured.** CrewAI's automatic task evaluation and memory saving only occurs when both memory systems are enabled. If you only configure `long_term_memory`, you'll need to save memories manually.
+
+## Prerequisites
+
+To use PostgreSQL as your long-term memory storage, you'll need to install the `postgres` extras:
+
+```bash
+pip install 'crewai[postgres]'
+```
+
+This will install the required `psycopg[pool]` dependency, which is a modern PostgreSQL client for Python with connection pooling support.
+
+**Note:** If you see warnings about "psycopg pool module not available", ensure you have the pool extras installed:
+
+```bash
+pip install 'psycopg[pool]'
+```
+
+## Configuration
+
+You can configure a crew to use PostgreSQL for long-term memory storage in several ways:
+
+### Option 1: Direct Storage Configuration
+
+```python
+from crewai import Agent, Crew, Task
+from crewai.memory.long_term.long_term_memory import LongTermMemory
+from crewai.memory.storage.ltm_postgres_storage import LTMPostgresStorage
+from crewai.memory.entity.entity_memory import EntityMemory
+
+# Initialize PostgreSQL storage
+postgres_storage = LTMPostgresStorage(
+    connection_string="postgresql://username:password@hostname:5432/database",
+    schema="public",                  # Optional, defaults to "public"
+    table_name="long_term_memories",  # Optional, defaults to "long_term_memories"
+    use_connection_pool=True,         # Optional, defaults to True
+    min_pool_size=1,                  # Optional, defaults to 1
+    max_pool_size=5                   # Optional, defaults to 5
+)
+
+# Create long-term memory with the PostgreSQL storage
+long_term_memory = LongTermMemory(storage=postgres_storage)
+
+# Create a crew with the configured memory (including entity memory for automatic saving)
+crew = Crew(
+    agents=[...],
+    tasks=[...],
+    memory=True,                    # Enable memory system
+    long_term_memory=long_term_memory,  # Use PostgreSQL for long-term memory
+    entity_memory=EntityMemory()    # Required for automatic memory saving
```

Comment:
For your use case, I'd recommend encouraging users to set only long_term_memory instead of configuring all the individual memory attributes.